tests/all_mode/run.sh

```diff
@@ -6,6 +6,7 @@ cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 source $cur/../_utils/test_prepare
 WORK_DIR=$TEST_DIR/$TEST_NAME
 API_VERSION="v1alpha1"
+ILLEGAL_CHAR_NAME='t-Ë!s`t'
```

Comment:
Seems it is a legal name

Reply:
yes, want to express ' not a valid name if not quoted in mysql' 🤣